### DISCUSSION TITLE
Forecast from API gridpoint wind speed units

### DISCUSSION THREAD

**Discussion by dmtreff** [weather-gov/api#787] [General]
I'm using the weather.gov API to pull a gridpoint forecast. The API call I am making is: https://api.weather.gov/gridpoints/LWX/113,87/forecast

The forecasted wind speeds are listed as MPH. Is there any option I can add to the API call to request the wind speed in the daily forecasts to be represented in Knots?

**Comment by dwhitemv25** [comment]
You can request the units for the entire forecast to be returned in either US customary (mph for wind) or SI (km/h for wind) units. There is no way to request wind units in knots as currently documented.
